### PR TITLE
Bump Enzyme convention

### PR DESCRIPTION
**Pull request by wsmoses**
This needs to correspond with using a new build of Enzyme proper.

### CODE REVIEW

**Comment by vchuravy** [comment]
I think we need to update the Yggdrasil builds as well?

**Comment by wsmoses** [comment]
On it, https://github.com/JuliaPackaging/Yggdrasil/pull/2042

**Comment by vchuravy** [comment]
Maybe the binary is already following those conventions? Otherwise CI shouldn't pass.

**Comment by wsmoses** [comment]
That's what I'm guessing given https://github.com/wsmoses/Enzyme.jl/issues/21 but I don't see how that happened

**Comment by vchuravy** [comment]
Yeah I messed up and pushed an Enzyme_jll without incrementing the patch version https://travis-ci.com/github/wsmoses/Enzyme.jl/jobs/427340400

**Comment by vchuravy** [comment]
needs https://github.com/JuliaRegistries/General/pull/24115

File: Project.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 
 [compat]
 Cassette = "0.3"
-Enzyme_jll = "0.0.1"
+Enzyme_jll = "0.0.2"
```
Comment (vchuravy) [inline Project.toml:15]:
```suggestion
Enzyme_jll = "0.0.3"
```

**Pull request merged**